EarlGrey/Provider/GREYElementProvider.m

```diff
@@ -149,8 +149,9 @@ - (NSEnumerator *)dataEnumerator {
               // Replace mock views with the views they encapsulate.
               element = [element view];
             }
-            NSAssert(element, @"element should not be nil");
-            [runningElementHierarchy addObject:element];
+            if(element) {
+              [runningElementHierarchy addObject:element];
+            }
```

Comment:
Could you also add a comment as to why we're doing this?
Also, you could have the same check in GREYElementHierarchy.m:137 - 

      // Need to add a nil check since sometimes accessibilityElement is nil.
      if (accessibilityElement) {
        [subViewSet addObject:accessibilityElement];
      }